README.md

```diff
@@ -1335,7 +1447,10 @@ This is because Mutation Observers queue a [microtask](https://jakearchibald.com
 
 ## Web Component Differences
 
-Skate implements the [Custom Element spec](http://w3c.github.io/webcomponents/spec/custom/) with a custom API but it does not polyfill the native methods. Since Skate is a custom element library, it does not polyfill [ShadowDOM](http://w3c.github.io/webcomponents/spec/shadow/) or [HTML Imports](http://w3c.github.io/webcomponents/spec/imports/).
+Skate implements the [Custom Element spec](http://w3c.github.io/webcomponents/spec/custom/) with a custom API but it does not polyfill the native methods. Since Skate requires that you BYO your own Shadow DOM polyfill. Some polyfills are:
```

Comment:
Needs revising. Should highlight: 

- it internally polyfills custom elements but not others and reasoning behind it (custom elements can be provided without touching native methods, shadow dom cannot).
- faster than webcomponentsjs custom element polyfill
- about the same size as custom element polyfill
- far more functionality